website/docs/reference/global-configs.md

```diff
@@ -269,6 +269,27 @@ config:
 
 </File>
 
+<VersionBlock firstVersion="1.2">
+
+### Project paths
+
+By default, dbt will write logs to a directory named `logs/` and all other artifacts to a directory named `target/`, both as relative paths to `dbt_project.yml` of the active project—that is, the root directory from which dbt is run.
+
+Unlike other global configs, project paths are configured in `dbt_project.yml` rather than the `config:` block in `profiles.yml`:
```

Comment:
remind me why again?

Reply:
that's just how it is, buster!

jk I'll add another sentence here